sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/MethodOverloadsShouldBeGroupedBase.cs

```diff
@@ -48,10 +48,10 @@ protected MethodOverloadsShouldBeGroupedBase(System.Resources.ResourceManager re
 
         protected void CheckMembers(SyntaxNodeAnalysisContext c, IEnumerable<TMemberDeclarationSyntax> members)
         {
-            foreach (var misplacedNameSyntaxes in GetMisplacedOverloads(c, members))
+            foreach (var misplaced in GetMisplacedOverloads(c, members))
```

Comment:
pedantic: it would be nice to rename `misplaced` with `misplacedOverload` since it make the code more clear